app/assets/javascripts/app/views/help_view.js

```diff
@@ -61,9 +53,7 @@ app.views.Help = app.views.StaticContentView.extend({
       title_sharing: Diaspora.I18n.t( 'sharing.title' ),
       title_tags: Diaspora.I18n.t( 'tags.title' ),
       title_keyboard_shortcuts: Diaspora.I18n.t( 'keyboard_shortcuts.title' ),
-      title_miscellaneous: Diaspora.I18n.t( 'miscellaneous.title' ),
-      title_chat: Diaspora.I18n.t( 'chat.title' ),
-      chat_enabled: this.chatEnabled()
+      title_miscellaneous: Diaspora.I18n.t( 'miscellaneous.title' )
```

Comment:
- Identifier 'title_miscellaneous' is not in camel case.
- There should be no spaces inside this paren.
- Strings must use doublequote.